variables.tf

```diff
@@ -25,8 +25,19 @@ variable "members" {
   default     = []
 
   validation {
-    condition     = alltrue([for m in var.members : can(regex("^(user|serviceAccount|group|domain):(.+)", m))])
-    error_message = "The value must be a non-empty list of strings where each entry is a valid principal type identified with a prefix such as e.g., `user:`, `serviceAccount:`, `group:` or `domain:`."
+    condition     = alltrue([for m in var.members : can(regex("^(allUsers|allAuthenticatedUsers|(user|serviceAccount|group|domain|computed):)", m))])
+    error_message = "The value must be a non-empty list of strings where each entry is a valid principal type identified with `allUsers`, `allAuthenticatedUsers` or prefixed with `user:`, `serviceAccount:`, `group:`, `domain:` or `computed`."
```

Comment:
i don't think allusers and allauthenticated is valid here... where did you get this from?

Reply:
https://github.com/mineiros-io/terraform-google-kms-key-ring-iam/blob/33428a62eddd07d60002b2a771311cc83ae0578a/variables.tf#L22

the oirigin :)

Reply:
yes but this is a different resource... with different features ofc ;) it wa snot there before.. so i  assume it is no available in this resource?

Reply:
@waxb why was this resolved but not addressed?